### PR TITLE
Interviews page

### PR DESCRIPTION
**Pull request by tuxella**

### CODE REVIEW

File: interview/templates/interview/base.html [hunk 1]
```diff
@@ -49,6 +49,7 @@
                         <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">{% trans "More" %} <span class="caret"></span></a>
                         <ul class="dropdown-menu">
                             <li><a href="{% url 'interviewers-load' %}">{% trans "Load" %}</a></li>
+			    <li><a href="{% url 'interviews-list' %}"> {% trans "All interviews" %} </a></li>
```
Comment (bport) [inline interview/templates/interview/base.html:52]: indentations are not consistent

File: interview/views.py [hunk 2]
```diff
@@ -178,6 +178,27 @@ class Meta:
...
 
 
+class InterviewDetailTable(InterviewTable):
+    process_detail = tables.TemplateColumn(
+        verbose_name=_("Candidate"), orderable=False, template_name="interview/tables/interview_process.html"
+    )
+
+    class Meta(InterviewTable.Meta):
+        sequence = (
+            "needs_attention",
```
Comment (bport) [inline interview/views.py:188]: Perhaps we can add itw rank ?

**Review by bport — CHANGES_REQUESTED** [review]

File: interview/views.py (hunk 2 above)
Comment (tuxella) [reply interview/views.py:190]: Added

File: interview/templates/interview/base.html (hunk 1 above)
Comment (tuxella) [reply interview/templates/interview/base.html]: Fixed: tabs replaced by spaces

**Pull request merged**